components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/add.jsp

```diff
@@ -444,11 +445,20 @@
                                 <tr>
                                     <td class="leftCol-med"><fmt:message key='oauth.version'/><span
                                             class="required">*</span></td>
-                                    <td><input id="oauthVersion10a" name="oauthVersion" type="radio"
-                                               value="<%=OAuthConstants.OAuthVersions.VERSION_1A%>"/>1.0a
+                                    <td>
+                                    <%
+                                    boolean OAuth1FieldDisabled = false;
+                                    boolean isOAuth1Enabled = Boolean.parseBoolean(IdentityUtil.getProperty("LegacyFeatures.EnableOAuth1"));
+                                    if (isOAuth1Enabled) {
+                                        OAuth1FieldDisabled = true;
```

Comment:
why we set OAuth1FieldDisabled to true when OAuth1 is enabled?

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1407/commits/1f613f565281710a85847663e9e82f1ed21e34db